commit 619f589f523448fd5b47f3c59ec2994642312585
Author: Theo Chatzimichos <tampakrap@gentoo.org>
Date:   Fri Feb 3 15:51:10 2012 +0100

    Fix optional PAM support, patch by Michael Palimaka <kde@scribeofthenile.com>
    
    REVIEW:103818

diff --git a/cmake/modules/UnixAuth.cmake b/cmake/modules/UnixAuth.cmake
index 904f0cb..0347251 100644
--- a/cmake/modules/UnixAuth.cmake
+++ b/cmake/modules/UnixAuth.cmake
@@ -1,4 +1,4 @@
-find_package(PAM)
+macro_optional_find_package(PAM)
 
 include(CheckFunctionExists)
 include(CheckLibraryExists)
